Havit.Blazor.Components.Web.Bootstrap/Forms/Internal/HxInputDateInternal.razor

```diff
@@ -61,7 +61,7 @@
                         {
                             foreach (var item in PredefinedDatesEffective)
                             {
-                                <HxButton @key="@item" Text="@StringLocalizerFactory.GetLocalizedValue(item.Label, item.ResourceType)" Color="ThemeColor.Link" Size="ButtonSize.Small" OnClick="() => HandleCustomDateClick(item.Date)" tabindex="-1" />
+                                <HxButton @key="@item" Text="@StringLocalizerFactory.GetLocalizedValue(item.Label, item.ResourceType)" Color="ThemeColor.Link" Size="ButtonSize.Small" OnClick="() => HandleCustomDateClick(item.Date ?? item.DateSelector(TimeProviderEffective))" tabindex="-1" />
```

Comment:
We should probably add some logic to prevent setting both `Date` and `DateSelector`.
Maybe a method `item.GetDateEffective()` which will check the properties and throw an exception when both are set?